Bottom: c6f4c2828412102a8ec5cf6fdc4be0df2beadd4f
Top:    4d9be6e67916049f264b6ad240bc8c800c5bfa1d
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 15:41:59 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 4a488e7..4aaeac3 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2177,7 +2177,8 @@ int main(int argc, char *argv[])
 
 		lookup_nss_read_master(master_list, 0);
 		master_show_mounts(master_list, dump);
-		free(dump);
+		if (dump)
+			free(dump);
 		exit(0);
 	}
